swagger.json

```diff
@@ -889,7 +889,62 @@
           }
         }
       }
-    }
+		},
+		"/api/v1/articles/bookmarks": {
+			"parameters": [{
+				"name": "token",
+				"in": "headers",
+				"required": true,
+				"description": "Authorization header",
+				"type": "string"
+			}],
+			"post": {
+				"summary": "Allows users to bookmark an article",
+				"description": "",
+				"schema": {
+					"$ref": "#/definitions/Bookmarks"
+				},
+				"responses": {
+					"200": {
+						"description": "Add articles to bookmark"
+					}
```

Comment:
**Good** Job Daniel
However, I think it will be nice to also document the outcome or interpretations of other possible status responses e.g `401`, `400`, `500`. Also, For a post request, the status code of `201` is preferred to `200` on successful creation.

Reply:
Thanks 👍